.changes/v1.0.0/57-notes.md

```diff
@@ -0,0 +1 @@
+* Add [`TROUBLESHOOTING.md`](./TROUBLESHOOTING.md) document to help diagnosing common issues [GH-57]
```

Comment:
Link points to `./TROUBLESHOOTING.md` as this will be in the root CHANGELOG.md file.